app/views/users/show.html.erb

```diff
@@ -18,6 +18,9 @@
       <%= "<span class=\"user_label label label-success\">Super User</span>".html_safe if @user.role == 'superuser' %>
       <%= "<span class=\"user_label label label-default\">Checkout Person</span>".html_safe if @user.role == 'checkout' %>
     <% end %>
+    <% if @has_pending %>
```

Comment:
No I don't think so. `can? :manage, User` is restricted to admins and above, but we want the badge to be visible to checkout persons / staff. We don't have to worry about restricting it to that role, though, since patrons can only view their own profiles (and they know about their own requests), so the only people who are viewing other users' profiles are checkout persons and above. Does that make sense?